firefox-ios/firefox-ios-tests/Tests/XCUITests/ActivityStreamTest.swift

```diff
@@ -32,8 +32,8 @@ class ActivityStreamTest: BaseTestCase {
                                LaunchArguments.SkipContextualHints,
                                LaunchArguments.DisableAnimations]
         }
-        launchArguments.append(LaunchArguments.SkipAddingGoogleTopSite)
-        launchArguments.append(LaunchArguments.SkipSponsoredShortcuts)
+        // launchArguments.append(LaunchArguments.SkipAddingGoogleTopSite)
+        // launchArguments.append(LaunchArguments.SkipSponsoredShortcuts)
```

Comment:
I don't know if these LaunchArguments are not supposed to work after the homepage redesign.